File: 3.Vérifier.go
package main

import (
	"fmt"
	"net/http"
	"time"
)

func main() {
	url := "https://example.com"
	client := http.Client{Timeout: 5 * time.Second}
	resp, err := client.Get(url)
	if err != nil {
		fmt.Println("Erreur de connexion")
		return
	}
	defer resp.Body.Close()
	if resp.StatusCode == 200 {
		fmt.Println("Site accessible")
	} else {
		fmt.Println("Site inaccessible", resp.StatusCode)
	}
}

File: 3.Vérifier.py
import requests

url = "https://example.com"
try:
    r = requests.get(url, timeout=5)
    if r.status_code == 200:
        print("Site accessible")
    else:
        print("Site inaccessible", r.status_code)
except:
    print("Erreur de connexion")
